src/katgpucbf/fgpu/recv.py

```diff
@@ -262,6 +262,8 @@ def make_sensors(sensor_timeout: float) -> aiokatcp.SensorSet:
                 "The timestamp (in UNIX time) when missing data was last detected",
                 default=aiokatcp.core.Timestamp(-1.0),
                 initial_status=aiokatcp.Sensor.Status.NOMINAL,
+                auto_strategy=aiokatcp.SensorSampler.Strategy.EVENT_RATE,
+                auto_strategy_parameters=(MIN_SENSOR_UPDATE_PERIOD, math.inf),
```

Comment:
I'm battling to find documentation which will help me understand why the second parameter is `math.inf` - what does that mean?

Reply:
It's in the katcp spec: it controls the maximum time between transmitting updates if the value hasn't changed. math.inf means that if the value doesn't change it just won't send updates.

Reply:
Ah. I was looking in the aiokatcp docs, I couldn't find it there. I didn't think to look in the spec.

Thanks.